package.json

```diff
@@ -57,7 +57,14 @@
   "sideEffects": false,
   "scripts": {
     "prebuild": "shx rm -rf dist",
-    "build": "rollup -c && rollup -c --config-utils && rollup -c --config-devtools && rollup -c --config-immer && rollup -c --config-optics && rollup -c --config-query && rollup -c --config-xstate",
+    "build": "concurrently 'yarn build:base' 'yarn build:utils' 'yarn build:devtools' 'yarn build:immer' 'yarn build:optics' 'yarn build:query' 'yarn build:xstate'",
```

Comment:
Can we do something like `build:*`?
https://www.npmjs.com/package/concurrently 👀

Reply:
nice! i didn't know about that, much better.